context/app/templates/pages/search.html

```diff
@@ -17,6 +17,111 @@ <h1 class="no-content">{% block title_text %}Search{% endblock %}</h1>
 <script src="https://unpkg.com/@hubmap/portal-search@v0.0.4/umd/@hubmap/portal-search.min.js"></script>
 
   <script>
+    const commonFilters = [
+      {
+        type: 'RefinementListFilter',
+        props: {
+          id: 'created_by_user_displayname',
+          title: 'Creator',
+          field: 'created_by_user_displayname.keyword',
+          operator: 'OR',
+          size: 5
+        }
+      },
+      {
+        type: 'RefinementListFilter',
+        props: {
+          id: 'contains_human_genetic_sequences',
+          title: 'Contains genetic sequences',
+          field: 'contains_human_genetic_sequences.keyword',
+          operator: 'OR',
+          size: 5
+        }
+      },
+      {
+        type: 'RefinementListFilter',
+        props: {
+          id: 'data_types',  // Not working?
+          title: 'Data type',
+          field: 'data_types.keyword',
+          operator: 'OR',
+          size: 5
+        }
+      },
+      {
+        type: 'RefinementListFilter',
+        props: {
+          id: 'donor_created_by',
+          title: 'Donor created by',
+          field: 'donor.created_by_user_displayname.keyword',
+          operator: 'OR',
+          size: 5
+        }
+      },
+      {
+        type: 'RefinementListFilter',
+        props: {
+          id: 'donor_group',
+          title: 'Donor group',
+          field: 'donor.group_name.keyword',
+          operator: 'OR',
+          size: 5
+        }
+      },
+      {
+        type: 'RefinementListFilter',
+        props: {
+          id: 'entity_type',
+          title: 'Type',
+          field: 'entity_type.keyword',
+          operator: 'OR',
+          size: 5
+        }
+      },
+      {
+        type: 'RefinementListFilter',
+        props: {
+          id: 'origin_sample_created_by',
+          title: 'Organ created by',
+          field: 'origin_sample.created_by_user_displayname.keyword',
+          operator: 'OR',
+          size: 5
+        }
+      },
+      {
+        type: 'RefinementListFilter',
+        props: {
+          id: 'source_sample_specimen_type',
+          title: 'Specimen type',
+          field: 'source_sample.specimen_type.keyword',
+          operator: 'OR',
+          size: 5
+        }
+      },
+      {
+        type: 'RefinementListFilter',
+        props: {
+          id: 'status',
+          title: 'Status',
+          field: 'status.keyword',
+          operator: 'OR',
+          size: 5
+        }
+      }
+    ];
+    const filtersByType = {
+      '': commonFilters,
+      donor: commonFilters,
+      sample: commonFilters,
+      dataset: commonFilters
+    };
+    const resultFieldsByType = {
+      '': ['description', 'status', 'entity_type'],
```

Comment:
Is there a reason why the fields you chose to display in the table are these ones? I was asking because it seems there's a lack of description for a lot of the entries and that status seemed to only apply to datasets.

Reply:
No reason at all. Please do find a better set of fields!